src/DotNet.Testcontainers.Tests/Unit/Containers/Unix/Database/CouchbaseTestcontainerTest.cs

```diff
@@ -0,0 +1,157 @@
+namespace DotNet.Testcontainers.Tests.Unit.Containers.Unix.Database
+{
+  using System;
+  using System.Threading.Tasks;
+  using Couchbase;
+  using Testcontainers.Containers.Builders;
+  using Testcontainers.Containers.Configurations.Databases;
+  using Testcontainers.Containers.Modules.Databases;
+  using Xunit;
+
+  public class CouchbaseTestcontainerTest
+  {
+
+    /// <summary>
+    /// Sample Couchbase document class for test
+    /// </summary>
+    private class Customer
+    {
+      public string Name { get; set; }
+      public int Age{ get; set; }
+    }
+
+    [Fact]
+    public async Task ConnectionEstablished_WithCrudTest()
+    {
+      //Given
+      var couchbaseTestcontainer = new TestcontainersBuilder<CouchbaseTestcontainer>()
+        .WithDatabase(new CouchbaseTestcontainerConfiguration("Administrator", "password", "Customer"))
+        .WithExposedPort(8091)
+        .WithExposedPort(11210)
+        .WithPortBinding(8091, 8091)
+        .WithPortBinding(11210, 11210)
+        .Build();
+
+      await couchbaseTestcontainer.StartAsync();
+
+      //When & Then
+      var cluster = await Cluster.ConnectAsync(couchbaseTestcontainer.ConnectionString, "Administrator","password");
+      var bucket = await cluster.BucketAsync("Customer");
+      var collection = bucket.DefaultCollection();
+
+      // Create
+      await collection.InsertAsync(
+        "customer-id-1",
+        new Customer {Name = "Mustafa", Age = 29}
+        );
+
+      // Read
+      var customer = (await collection.GetAsync("customer-id-1")).ContentAs<Customer>();
+      Assert.True(customer.Name == "Mustafa");
```

Comment:
Use `Assert.Equal("Mustafa", customer.Name);` instead. Use this for the other assertions too please.

Reply:
Sorry about that. I'm not familiar to use Assert functions over C# because I'm using FluentAssertions. I'll update.